### PR TITLE
[WIP] White font for place type search box and x

### LINKED ISSUE #3236: White font for place type search box and "x"
I could not find 😂
It hides part of the map.
Screenshot_20191129-185633_Commons

### PR DESCRIPTION
**Pull request by chaytanyasinha**
Description
This is solution for issue #3236

Fixes #3236

What changes did you make and why?
The hints and "X" was not visible in black color.

Tests performed
I tried to test but it was showing error of sdk files not found.

### CODE REVIEW

**Comment by neslihanturan** [comment]
Hey @chaytanyasinha , thanks a lot. Can you please add screenshoots since this PR effects UI

**Comment by chaytanyasinha** [comment]
I would have definitely added it but it was showing error of "sdk files"and because of my ram problem I am not able to update my Android studio.
Sorry

**Comment by neslihanturan** [comment]
Unfortunately this does not work @chaytanyasinha , and I see some problems on the code too. Will be easier to solve the issue in another PR than fixing this one, considering your ram situation. Thanks anyway!

File: app/src/main/java/fr/free/nrw/commons/nearby/NearbyController.java [hunk 1]
```diff
@@ -26,6 +26,9 @@
 
 import static fr.free.nrw.commons.utils.LengthUtils.computeDistanceBetween;
 import static fr.free.nrw.commons.utils.LengthUtils.formatDistanceBetween;
+import com.actionbarsherlock.widget.SearchView;
+import com.actionbarsherlock.widget.SearchView.SearchAutoComplete;
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/nearby/NearbyController.java]: Why to import these two?

File: app/src/main/res/drawable/ic_clear_white_24dp.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#FFFFFF"
```
Comment (neslihanturan) [inline app/src/main/res/drawable/ic_clear_white_24dp.xml]: This file is not used in anywhere, so won't have any effect. 
Additionally ic_clear_black_24dp is used in answer_layout.xml at line 62, so we can't delete it. Even if it wasn't being in use, we still can't delete it in an irrelevant PR.

**Comment by neslihanturan** [comment]
Sure @chaytanyasinha , I submitted my reviews for some major issues with this PR. Besides, I don't know if your solution will work or not. You don't know it either since you can't test the code. Since this Pr has some build issues too, and we are not sure about the solutions, and we don't have any screenshots o anything, rewriting this PR will be more time consuming then sending the fix myself. Additionally, for this specific issue we don't have that much time, we want it soon. This is why I closed your PR. I am hoping that you solve your issues with your SDK and continue to contribute soon.

**Comment by neslihanturan** [comment]
By the way thanks for following the description guidelines, this was a nice PR for a first time contributor:)

**Comment by chaytanyasinha** [comment]
Thanks @neslihanturan I'll soon try to resolve the issues.

**Pull request closed without merging**